.vscode/spellchecker.json

```diff
@@ -0,0 +1,45 @@
+{
+    "ignoreWordsList": [
+        "Brockenhurst",
+        "Basingstoke",
+        "Gordano",
+        "Guildford",
+        "Headington",
+        "Markt",
+        "Indersdorf",
+        "Mirfield",
+        "Tunbridge",
+        "prepend",
+        "baseurl",
+        "blueshirt",
+        "blueshirts",
+        "Kickstart",
+        "Kickstarts",
+        "KickStart",
```

Comment:
Is both lower case and upper case _'S'_ valid for _Kickstart_? Personally i'd only use lower